### ISSUE TITLE
The unsigned transaction format, used by cold wallets, may cause issues when moving between platforms (eg. from Linux to Windows) for signing.

### ISSUE THREAD

**Issue by medusadigital** [monero-project/monero#1979]
Opening Issue here instead of monero-core repo.

**Comment by NanoAkron** [comment]
Do you have an example/log messages demonstrating this issue?

**Comment by iamsmooth** [comment]
or failing that, maybe be a bit more specific about what you believe the problem to be?

**Comment by medusadigital** [comment]
The Issue text is copied from the monero-core GUI Beta 2 Release Notes (known issues). 

I opened it now as a place holder since i missed this issue. 

however, when digging deeper, i found the initial reason it was added to the release notes: https://forum.getmonero.org/5/support/87622/different-recipient-address-when-signing-transfer-offline

it seems that in the meantime this claim has been invalidated by @moneromooo-monero , sorry i didnt notice. 

however, follow up issue should probably be created, addressing the much smaller issue with the wrong confirmation message in the offline wallet (displaying a non integrated address). 


--> will close this issue again therefore.
